database/index.js

```diff
@@ -1,29 +1,33 @@
 const mongoose = require('mongoose');
-const { createProducts, createMiscPages } = require('./seed.js')
+const SimpleSchema =  require('simpl-schema');
+const mongodb = require('mongodb');
+const { createProducts, createMiscPages } = require('./seed.js');
 
 let url = 'mongodb://localhost/navbar'; // connect to local DB: "navbar"
-mongoose.connect(url, { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false});
+//mongoose.connect(url, { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false});
+mongodb.MongoClient.connect(url, { useNewUrlParser: true, useUnifiedTopology: true});
 
 let Schema = mongoose.Schema;
 
 // create product schema
 let productSchema = new Schema({
-  name: String,
-  price: Number,
-  img: String,
+  name: {type: String},
+  price: {type: Number},
+  img: {type: String},

```

Comment:
Was adding them as a type like this specifically how Mongo (without mongoose) wants it formatted? Did this help resolve errors you ran into or was it just preference?